src/components/editor/ResourceTemplateModal.jsx

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018 Stanford University see Apache2.txt for license
+
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import Button from 'react-bootstrap/lib/Button'
+import Modal from 'react-bootstrap/lib/Modal'
+import ResourceTemplateForm from './ResourceTemplateForm'
+
+class ResourceTemplateModal extends Component {
+  constructor(props) {
+    super(props)
+    this.handleClose = this.handleClose.bind(this)
+    this.handleSave = this.handleSave.bind(this)
+    this.saveDataFromModal = this.saveDataFromModal.bind(this)
+  }
+
+  handleClose = () => {
+    this.props.toggleVisibility()
+  }
+
+  handleSave = () => {
+    this.saveDataFromModal()
+    this.handleClose()
+  }
+
+  saveDataFromModal = () => {
+    console.log('TODO: implement holding data from ResourceTemplateModal')
```

Comment:
Should we create a ticket # for this?

Reply:
Yeah, I think this is a candidate for using Redux.